### ISSUE TITLE
[Feature] Routine load process with a exist group id 

### ISSUE THREAD

**Issue by learner1212** [apache/doris#18535]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Description

  routine load now not support modify columns, so when my table schema changed, we must stop current routine load job and submit a new one. but doris will reset offset to lastest or earilest by property.kafka_default_offsets setting, that may cause data repeat or data lost. the only way we avoid this problem is appoint kafka_partitions and kafka_offsets settings, and this also bring new problems:
1. the configuration is difficult when we have a large number of partitions;
2. when we increase partiton in topic, we will loss data in the new partitions.

in summary， we need to consume kafka topic with a exist consumer group an do not reset the offset.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by learner1212** [comment]
> Can we support this feature in future versions?

**Comment by learner1212** [comment]
> Sounds reasonable

Can we support this feature in future versions?

**Comment by dataroaring** [comment]
yep

**Comment by reswqa** [comment]
Thanks for reporting this, I'd like to help support this feature.
